### PR TITLE
softwarecontainer: Update version

### PR DESCRIPTION
**Pull request by socec**

### CODE REVIEW

File: recipes-core/softwarecontainer/softwarecontainer.bb [hunk 1]
```diff
@@ -30,11 +30,6 @@ PACKAGECONFIG ?= "networkgateway devicenodegateway dbusgateway cgroupsgateway"
 
 EXTRA_OECMAKE += "-DENABLE_TEST=OFF -DENABLE_EXAMPLES=OFF"
 
-do_install_append() {
```
Comment (jonte) [inline recipes-core/softwarecontainer/softwarecontainer.bb]: Isn't this needed anymore?

Comment (jonte) [inline recipes-core/softwarecontainer/softwarecontainer.bb]: Oh.. You now need to commit your review comments.. Sorry for this one coming in a bit late, I missed the button.

Comment (socec) [inline recipes-core/softwarecontainer/softwarecontainer.bb]: The install path is now handled in CMake to support both Yocto and local development. Therefor this workaround is not needed anymore.

Comment (jonte) [inline recipes-core/softwarecontainer/softwarecontainer.bb]: Ah, I see. Thanks!

**Pull request merged**